src/plugins/gpunetio/verbs/verbs.cpp

```
@@ -56,18 +64,57 @@ static uint32_t
+        if (destroy() != DOCA_SUCCESS) {
+            NIXL_ERROR << "Failed to destroy doca verbs cq attributes";
+        }
+    }
+};
+
+class QpAttrGuard {
+public:
+    doca_verbs_qp_init_attr *value = nullptr;
+
+    doca_error_t
+    destroy() noexcept {
```

Cppcheck (2.21.0)

[style] The function 'get_qp' is never used.

(unusedFunction)